build.gradle

```diff
@@ -5,9 +5,8 @@ import java.time.LocalDate
 
 plugins {
     id 'java'
-    id 'org.springframework.boot' version '2.2.2.RELEASE' // remove pinning of tomcat version to 9.0.30 when next upgrading
-                                                          // also remove pinning of nimbus-jose-jwt to 8.4 when next upgrading
-    id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+    id 'org.springframework.boot' version '2.2.4.RELEASE' // remove pinning of nimbus-jose-jwt to 8.4 when next upgrading
```

Comment:
The version of nimbus-jose-jwt that is pulled through by default still holds the vulnerability